qubesmanager/main.py

```diff
@@ -465,6 +466,9 @@ def __init__(self, qvm_collection, blk_manager, parent=None):
 
         self.connect(self.searchbox, SIGNAL("textChanged(const QString&)"),
                      self.do_search)
+        self.searchbox.focusInEvent = types.MethodType(
+            lambda w, e: QTimer.singleShot(0, self.searchbox.selectAll),
+            self.searchbox.focusInEvent)
```

Comment:
A timer is still required because the mousePressEvent gets handled after the focusInEvent, and the mousePressEvent deselects the text.